tests/responses/tests.py

```diff
@@ -63,10 +63,22 @@ def test_change_status_code(self):
         self.assertEqual(resp.status_code, 503)
         self.assertEqual(resp.reason_phrase, "Service Unavailable")
 
+    def test_valid_status_code_string(self):
+        resp = HttpResponse(status='403')
+        self.assertEqual(resp.status_code, 403)
+
+    def test_invalid_status_code(self):
+        with self.assertRaisesMessage(TypeError, 'HTTP status code must be an integer'):
+            HttpResponse(status='bender')
+        with self.assertRaisesMessage(ValueError, 'HTTP status code must be an integer from 100 to 599'):
+            HttpResponse(status=22)
```

Comment:
Will do. Was having too much fun 😝 
> Bender (Futurama)
From Wikipedia, the free encyclopedia
Bender Bending Rodríguez (designated in-universe as Bending Unit 22, unit number 1,729, serial number 2716057[1]) is a main character in the animated television series Futurama.